File: src/forms/requests/get_values.go
package requests

type GetValues struct {
	APIKey  string `form:"apiKey" url:"apiKey"`
	FieldID uint   `form:"fieldId" url:"fieldId"`
	Order   string `form:"order" url:"order"`
	Offset  int    `form:"offset" url:"offset"`
	Limit   int    `form:"limit" url:"limit"`
}

File: Packages/dotnet/Trackr/getValues.cs
namespace Trackr.Models
{
    public class getValues
    {
        public string ApiKey { get; set; }
        public string Value { get; set; }
        public uint Offset { get; set; }
        public int Limit { get; set; }
        public string Order { get; set; }
    }
}